### PR TITLE
V6/fixes network

### PR DESCRIPTION
**Pull request by olgakup**
### Devop

- This Pr ads balances to the address in mnemonic /hardware access
- hides unswappable networks when on swap route
- changes network selection in mnemonic and hardware wallets
- hides tarde button in token balances on networks that do not have swap

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

**Review by gesseekur — APPROVED** [review]

File: src/modules/balance/ModuleTokensValue.vue [hunk 1]
```diff
@@ -55,6 +63,8 @@ export default {
   data: () => ({ showPopup: false }),
   computed: {
     ...mapGetters('wallet', ['tokensList']),
+    ...mapState('wallet', ['loadingWalletInfo']),
```
Comment (kvhnuke) [inline src/modules/balance/ModuleTokensValue.vue:66]: You should still use initialLoad

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/modules/balance/ModuleTokensValue.vue (hunk 1 above)
Comment (olgakup) [reply src/modules/balance/ModuleTokensValue.vue:66]: fixed!

File: src/modules/network/components/NetworkSwitch.vue [hunk 2]
```diff
@@ -160,28 +149,42 @@ export default {
...
+        unsorted.splice(unsorted.indexOf('ETH'), 1);
+        unsorted.sort();
+        const test = unsorted.filter(item => {
+          return types[item].isTestNetwork;
+        });
+        const main = unsorted.filter(item => {
+          return !types[item].isTestNetwork;
+        });
+        const sorted = main.concat(test);
+        sorted.unshift('ETH');
```
Comment (kvhnuke) [inline src/modules/network/components/NetworkSwitch.vue:166]: what if `ETH` is already part of the list?

**Review by kvhnuke — APPROVED** [review]

File: src/modules/network/components/NetworkSwitch.vue (hunk 2 above)
Comment (olgakup) [reply src/modules/network/components/NetworkSwitch.vue:166]: it's just moving it to front

Comment (olgakup) [reply src/modules/network/components/NetworkSwitch.vue:166]: So it always in the front even after sorting

**Review by kvhnuke — APPROVED** [review]

**Pull request merged**